Converted Datadog_API.json to Datadog_API.jsonl

divijmuthu opened the pull request:
Hi,

I was able to complete this conversion in gorilla/data/apizoo locally using Jupyter Notebook and upload the jsonl file, wanted to make sure if I should follow this process for all the .json files in the dataset as well?

divijmuthu commented:
Hi, 

I was also able to convert most of the other .json files under gorilla/data/apizoo to .jsonl except for a couple with formatting issues (Gmail_API, KubernetesAPI, Kubernetes...PodTemplateAPI, Salesforce...REST Format).

divijmuthu commented:
Hi @ShishirPatil @HuanzhiMao, just wanted to check on this PR, I've also been looking through the bfcl folder and specifically files importing json, wanted to confirm which files are part of the bfcl pipeline that needs to be updated from json to jsonl such as berkeley-function-call-leaderboard/bfcl/eval_checker/executable_eval/executable_checker.py, berkeley-function-call-leaderboard/bfcl/eval_checker/eval_runner_helper.py, some of the models in berkeley-function-call-leaderboard/bfcl/model_handler/oss_model, etc.

HuanzhiMao commented:
> berkeley-function-call-leaderboard

Only the `berkeley-function-call-leaderboard` folder is relavant to the bfcl project

divijmuthu commented:
Hi, @ShishirPatil @HuanzhiMao, I have updated the python files (_apply_function_credential_config.py, etc) that read and wrote json files to handle jsonl files using the jsonlines library (https://jsonlines.readthedocs.io/en/latest/). I have also updated the files in the bfcl/data folder since they were validated as proper jsonl files by https://jsonlines.org/validator/ when I pasted their contents, so their extensions could be updated to jsonl to be consistent with the changes to the python files. Kindly review the changes in the PR, thanks so much.
